WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -16,17 +16,17 @@ def competition_message_for_user(competition, user)
   end
 
   def announced_class(days_announced)
-    level = [21, 28].select {|d| days_announced > d}.count
+    level = [Competition::ANNOUNCED_DAYS_WARNING, Competition::ANNOUNCED_DAYS_DANGER].select {|d| days_announced > d}.count
     ["alert-danger", "alert-orange", "alert-green"][level]
   end
 
   def report_class(days_report)
-    level = [7, 14, 21].select {|d| days_report > d}.count
+    level = [Competition::REPORT_AND_RESULTS_DAYS_OK, Competition::REPORT_AND_RESULTS_DAYS_WARNING, Competition::REPORT_AND_RESULTS_DAYS_DANGER].select {|d| days_report > d}.count
     ["alert-green", "alert-success", "alert-orange", "alert-danger"][level]
   end
 
   def results_class(days_results)
```

Comment:
It's confusing to me that we have a separate `report_class` and `results_class` method, especially in light of all the other `report_and_result` methods you just created. If `report_class` and `results_class` ever diverge, it's going to break apart a lot of other things you wrote here. I think that for consistency, we should rename this to `report_and_results_class` as well.